src/content/api/payment-requests.mdx

```diff
@@ -101,8 +101,11 @@ Centrapay Payment Requests are serviced via two sets of endpoints; the “next
   <Property name="invoiceRef" type="string">
     A reference to an invoice for this Payment Request. Must be less than or equal to 128 characters.
   </Property>
-  <Property name="redirectUrl" type="string" experimental>
-    URL to redirect the user to after they pay or cancel the Payment Request. Must start with one of the allowedRedirectUrls for the [Merchant Config](/api/merchant-configs).
+  <Property name="redirectCancelUrl" type="string" experimental>
```

Comment:
Have left these as experimental in case we change our mind later. We will be the only integrator with them initially